### ISSUE TITLE
Questionnaire : Afficher le graphique détaillé par défaut ?

### ISSUE THREAD

**Issue by bjlaa** [datagir/nosgestesclimat-site#926]
Actuellement le bouton pour afficher le graphique détaillé, me parait peut visible : il est de couleur très claire et n'a pas de label.

Ce graphique apporte des infos précieuses à l'utilisateur qui sont complémentaires de la bar graphique (affichée en permanence). Il mériterait d'être affiché par défaut selon moi, car il ajoute certes de la hauteur de scroll mais étant le dernier élément de la section je ne vois pas en quoi cela pourrait présenter problème.

**Comment by laem** [comment]
Il a justement été caché parce qu'on trouverait qu'il y avait trop d'informations sur la page :sweat_smile:. Je pense qu'il y a en effet trop d'info sur la page pendant le test si on ajoute ce troisième graphique. 

Le bouton effacé est intentionnel, c'est à considérer comme une fonctionnalité experte.

**Comment by bjlaa** [comment]
Je suis d'accord avec le fait qu'il faut rester le plus "mince" possible niveau UI ! En revanche, pour le bouton je trouve ça dommage de le "cacher", en l'état il est quasiment inutilisable et c'est quand même une chouette fonctionnalité pour ceux que ça intéresse. Je pense qu'un bouton de menu dropdown "Voir le détail" serait plus approprié pour deux raisons : 
* ce bouton serait de toute manière utilisé uniquement par les utilisateurs motivés à rechercher plus d'information et ne déteriorerait pas l'expérience pour les autres utilisateurs
* le bouton actuel n'est pas accessible

**Comment by laem** [comment]
Étant donné le peu d'informations supplémentaires que donne ce graphique (qui avait été codé avant l'autre, l'autre étant une simplification du premier), je ne suis pas convaincu par le fait de le mettre en avant. Je crains qu'un bouton "Voir le détail" conduirait beaucoup d'utilisateur à y voir une promesse non satisfaite. 

Actuellement, il ne fait qu'afficher les pourcentages, et les barres éventuellement cachés. Il a même moins d'informations que le coupe graphique de catégories + graphique de sous-catégories. Mais quand il y a des barres résiduelles cachées, elles déclenchent un bouton supplémentaire pour afficher ce graphique détaillé. 

![image](https://github.com/datagir/nosgestesclimat-site/assets/1177762/4e29d566-4223-403c-9a8d-3c57e9240823)

Donc pas contre afficher un bouton "voir le détail", mais il faudrait apporter à mon avis une valeur ajoutée plus importante.
